src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/dao/HttpGoogleBillingDAO.scala

```diff
@@ -487,6 +493,26 @@ class HttpGoogleBillingDAO(appName: String,
       }
     } yield {}
 
+  def cleanupGkeClusters(projectName: String): Future[Unit] =
+    for {
+      result <- googleRq(
+        gke.projects().zones().clusters().list(projectName, "us-central1-a")
+      )
+      gkeClusters = googNull(result.getClusters)
+      _ <- sequentially(gkeClusters) { gkeCluster =>
+        val nameString =
+          s"projects/${projectName}/locations/${gkeCluster.getLocation}/clusters/${gkeCluster.getName}"
+        logger info s"cluster name string: ${nameString}"
```

Comment:
should this be debug level?

Reply:
oh I actually meant to delete this info log. but for now i will change it to `debug` in case it's helpful when testing on dev